webui.py

```diff
@@ -2580,7 +2580,8 @@ def _validate_metadata_1_2(self, data):
         # so we can use its check() method
         metadata = Metadata()
         for key, value in data.items():
-            metadata[key] = value
+            if key not in ('blake2_256_digest', 'comment', 'filetype', 'md5_digest', 'protcol_version', 'pyversion', 'sha256_digest'):
```

Comment:
There is a chance that I'm missing something obvious, but shouldn't "protcol" be "protocol"?